easybuild/easyconfigs/c/CP2K/fix_compile_date_lastcvs.patch

```diff
@@ -1,14 +1,11 @@
---- makefiles/Makefile.orig	2010-12-10 05:35:15.000000000 +0100
-+++ makefiles/Makefile	2011-01-24 10:14:35.000000000 +0100
-@@ -149,9 +149,9 @@
+--- makefiles/Makefile.orig	2012-08-21 14:35:17.000000000 +0300
++++ makefiles/Makefile	2013-11-12 12:41:29.361582641 +0200
+@@ -194,7 +194,7 @@
  #
  ifeq ($(CPPSHELL),)
  CPPSHELL := -D__COMPILE_ARCH="\"$(ARCH)\""\
 -            -D__COMPILE_DATE="\"$(shell date)\""\
 +            -D__COMPILE_DATE="\"$(shell date | sed 's/ /_/g')\""\
```

Comment:
this basically means this patch is very specific to this CP2K version, so rename it to `CP2K-2.4.0_fix...`?